sample-kotlin/src/main/java/com/fblitho/lithoktsample/errors/ListRowComponentSpec.kt

```diff
@@ -25,17 +28,17 @@ object ListRowComponentSpec {
 
   @OnCreateLayout
   fun onCreateLayout(c: ComponentContext, @Prop row: ListRow): Component =
-      Column.create(c)
-          .paddingDip(YogaEdge.VERTICAL, 8f)
-          .paddingDip(YogaEdge.HORIZONTAL, 32f)
-          .child(
-              Card.create(c)
-                  .content(
-                      Column.create(c)
-                          .marginDip(YogaEdge.ALL, 32f)
-                          .child(TitleComponent.create(c).title(row.title))
-                          .child(PossiblyCrashingSubTitleComponent.create(c).subtitle(row.subtitle))
-                          .build())
-                  .build())
-          .build()
+      column(c) {
+        paddingDip(YogaEdge.VERTICAL, 8f)
+        paddingDip(YogaEdge.HORIZONTAL, 32f)
+        children {
+          card {
+            content(column(c) {
+              marginDip(YogaEdge.ALL, 32f)
+              child(TitleComponent.create(c).title(row.title))
```

Comment:
Its not too great to have to resort back to the builder for your own components imo... Perhaps we need some sort of factory method that returns the builder in a lambda. Then again, it seems like you would just do `MyComponent.create(c).apply { ... }`